src/CoreWCF.WebHttp/tests/OpenApiWcfTests.cs

```diff
@@ -1089,7 +1090,7 @@ public void PropertyCanBeHiddenWithTag()
         private class SimplePropertyClass
         {
             [DataMember(Name = "Property")]
-            [OpenApiProperty(Description = "description", IsRequired = true)]
+            [OpenApiProperty(Description = "description", IsRequired = true, MinLength = 1, MaxLength = 2, Format = "format")]
             public string Property { get; set; }
```

Comment:
Is there a negative test to make sure that when MinLength and MaxLength are not set, that they aren't in the generated json file? I couldn't find one, but it's possible it's implicitly tested somewhere.

Reply:
I added some additional asserts for the negative case.